core/src/main/scala/ammrunner/AmmoniteFetcher.scala

```diff
@@ -59,23 +59,23 @@ import scala.io.{BufferedSource, Codec}
         if (interpOnly)
           "ammonite-interp-api_" + versions.scalaVersion
         else
-          "ammonite-repl-api_" + versions.scalaVersion,
+          "ammonite-repl-api_" + twoThreeInterop(versions.scalaVersion),
```

Comment:
It's still a `2.13` artifact in that case, yes, but its name is slightly different. It should be [`ammonite-cross-23-repl-api`](https://repo1.maven.org/maven2/com/lihaoyi/ammonite-cross-23-repl-api_2.13.6/) (this one pulls slightly different dependencies, favoring the `_3` ones in particular).

Reply:
Sounds good. Switched this to use that then if it's scala 3.